### PR TITLE
fix: SDA-1857 for us keyboard we should also zoom in for command and =

### PR DESCRIPTION
**Pull request by johankwarnmarksymphony**
https://perzoinc.atlassian.net/browse/sda-1857 Shortcuts: Zoom in hot keys not functional (Mac only)

Add so US keyboard can zoom in with "Command" and "="

### CODE REVIEW

File: src/app/window-handler.ts [hunk 1]
```diff
@@ -1006,6 +1011,17 @@ export class WindowHandler {
         reloadWindow(focusedWindow as ICustomBrowserWindow);
     }
 
+    /**
+     * Zoom in
+     */
+    private onZoomIn(): void {
```
Comment (KiranNiranjan) [inline src/app/window-handler.ts]: @johankwarnmarksymphony Is this a workaround? do we have an open issue in electron? if so, can you please link to this PR? Thanks

File: src/app/window-handler.ts [hunk 2]
```diff
@@ -1006,6 +1011,17 @@ export class WindowHandler {
...
+        const focusedWindow = BrowserWindow.getFocusedWindow();
+        if (!focusedWindow || !windowExists(focusedWindow)) {
+            return;
+        }
+        focusedWindow.webContents.zoomLevel += 0.5;
```
Comment (mattias-symphony) [inline src/app/window-handler.ts]: Use a named constant for this? And why 0.5 specifically?

**Comment by VishwasShashidhar** [comment]
@johankwarnmarksymphony We can use [this solution](https://github.com/electron/electron/issues/15496#issuecomment-450685872) instead of writing custom code.

There's an open issue in Electron for this issue @KiranNiranjan , check the link above.

File: src/app/window-handler.ts (hunk 1 above)
Comment (johankwarnmarksymphony) [reply src/app/window-handler.ts]: Yes, we are already using the accelerator
zoomIn: 'Ctrl+=',
This accelerator works on windows but not on mac.
So on mac we need to listen for Command/= too

File: src/app/window-handler.ts (hunk 2 above)
Comment (johankwarnmarksymphony) [reply src/app/window-handler.ts]: From Electron source code
zoomin: {
    label: 'Zoom In',
    accelerator: 'CommandOrControl+Plus',
    nonNativeMacOSRole: true,
    webContentsMethod: (webContents) => {
      webContents.zoomLevel += 0.5
    }
  },

So we zoom with same level as Command/+

**Comment by johankwarnmarksymphony** [comment]
Better solution will come from @KiranNiranjan

**Pull request closed without merging**